courses/management/commands/manage_program_certificates.py

```diff
@@ -0,0 +1,81 @@
+"""
+Management command to create program certificate(s)
+
+Arguments:
+* --user <username/ email> - an email or a username for a user to generate certificate for
+* --program <readable_id> - Program readable_id for certificate generation
+
+You must specify --program, since that will be used to filter programs for certificate generation
+"""
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+
+from courses.models import CourseRunEnrollment
+from courses.utils import generate_program_certificate
+from users.api import fetch_user
+
+
+class Command(BaseCommand):
+    """
+    Command to create program certificate for users.
+    """
+
+    help = "Create program certifificate, specific or all, for user(s)."
```

Comment:
Re-doing this comment for the typo.
```suggestion
    help = "Create program certificate, for a single user or all users in the program."
```